### PR TITLE
have cause of SQLServerException exception at any place where it possible

### PR DESCRIPTION
**Pull request by v-xiangs**
for issue #139

### CODE REVIEW

File: src/test/java/com/microsoft/sqlserver/jdbc/exception/ExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        }
+        finally {
+            if (null != conn) {
+                conn.close();
+            }
+        }
+    }
+
+    private void dropWaitForDelayProcedure(SQLServerConnection conn) throws SQLException {
+        String sql = " IF EXISTS (select * from sysobjects where id = object_id(N'" + waitForDelaySPName
```
Comment (v-nisidh) [inline src/test/java/com/microsoft/sqlserver/jdbc/exception/ExceptionTest.java]: Use new Utility for delete object. `Utils.dropObjectIfExists(...)`

File: src/test/java/com/microsoft/sqlserver/jdbc/exception/ExceptionTest.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            assertTrue(null != e.getCause(), "Cause should not be null.");
+            assertTrue(e.getCause() instanceof UnsupportedEncodingException, "Cause should be instance of UnsupportedEncodingException.");
+        }
+    }
+
+    /**
+     * 
+     * @return location of resource file
+     */
+    static String getCurrentClassPath() {
```
Comment (v-nisidh) [inline src/test/java/com/microsoft/sqlserver/jdbc/exception/ExceptionTest.java]: Can you move this to Util class as it is very much common to whole framework. 

@v-suhame : Possibly we already might be having this functionality of `getCurrentClassPath` somewhere.

Comment (v-xiangs) [inline src/test/java/com/microsoft/sqlserver/jdbc/exception/ExceptionTest.java]: we have `getCurrentClassPath()` in BulkCopyCSVTest as well. I moved them into Utils class.

File: src/test/java/com/microsoft/sqlserver/jdbc/exception/ExceptionTest.java (hunk 1 above)
Comment (v-xiangs) [inline src/test/java/com/microsoft/sqlserver/jdbc/exception/ExceptionTest.java]: This PR is pretty old, before we have this `Utils.dropObjectIfExists(...)` method.

Comment (v-xiangs) [inline src/test/java/com/microsoft/sqlserver/jdbc/exception/ExceptionTest.java]: changed to `Utils.dropProcedureIfExists` after merging/updating with Dev branch

**Pull request merged**